go/service/tlf.go

```diff
@@ -58,3 +61,45 @@ func (h *tlfHandler) CompleteAndCanonicalizePrivateTlfName(ctx context.Context,
 	ctx = chat.Context(ctx, arg.IdentifyBehavior, &breaks, chat.NewIdentifyNotifier(h.G()))
 	return h.tlfInfoSource.CompleteAndCanonicalizePrivateTlfName(ctx, arg.TlfName)
 }
+
+func (h *tlfHandler) identifyUser(ctx context.Context, assertion string, private bool, idBehavior keybase1.TLFIdentifyBehavior) (keybase1.TLFIdentifyFailure, error) {
```

Comment:
It seems to me like we never actually want to use the UI for anything for chat identifies, since we do something completely different for displaying the results. Maybe we could get a speed win by just doing nothing on all those calls?